ui/src/components/TopContainer/Header.tsx

```diff
@@ -51,13 +58,69 @@ const useStyles = makeStyles((theme: Theme) =>
         cursor: 'pointer',
       },
     },
+    searchTrigger: {
+      borderRadius: '4px',
+      color: '#969faf',
+      cursor: 'pointer',
+      '&:hover': {
+        color: '#1c1e21',
+      },
+    },
+    searchModal: {
+      position: 'relative',
+      maxWidth: '35rem',
+      minHeight: '12.8125rem',
+      margin: '3.75rem auto auto',
+      padding: 12,
+      background: '#f5f6f7',
+      outline: 0,
+    },
   })
 )
 
 function hasLocalBreadcrumb(b: string) {
   return ['overview', 'experiments', 'newExperiment', 'events', 'archives', 'settings'].includes(b)
 }
 
+const SearchTrigger: React.FC = () => {
```

Comment:
Encapsulate this component into a file?

Also, add a global shortcut to trigger searching? like `Ctrl/CMD + P`.